packages/datatrak-web/src/features/Tasks/TasksTable/FilterToolbar.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { useQueryClient } from 'react-query';
+import {
+  FormGroup as MuiFormGroup,
+  FormControlLabel as MuiFormControlLabel,
+  Checkbox as MuiCheckbox,
+} from '@material-ui/core';
+import { getTaskFilterSetting, setTaskFilterSetting } from '../../../utils';
+import { TaskFilterType } from '../../../types';
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 0.1rem 0 0;
+  border-bottom: 1px solid ${({ theme }) => theme.palette.divider};
+`;
+
+const FormGroup = styled(MuiFormGroup)`
+  display: flex;
+  flex-direction: row;
+`;
+
+const FormControlLabel = styled(MuiFormControlLabel)`
+  .MuiFormControlLabel-label {
+    font-size: 0.625rem;
+    color: ${({ theme }) => theme.palette.text.secondary};
+    padding-right: 0.8rem;
```

Comment:
```suggestion
    padding-inline-end: 0.8rem;
```

We kind of started using `*-inline-*` and `*-block-*` with padding and margins lately, because it works for `rtl` locales too. Not strictly essential but kind of nice to have in case we translate things one day